### PR TITLE
Fixed statement typo at the end of the readme.md file

### PR DESCRIPTION
**Pull request by immatobi**
#### Readme.md statement fix
I noticed the `contributing` part of the `readme.md` file says `Your pull request are definitely welcome ❤️! We alway looking to add more test and acheive feature parity with the Buycoins API...` all i did here is to change `We alway` to `We're always` and the new statement looks like this: `Your pull request are definitely welcome :heart:! We're always looking forward to add more test and acheive feature parity with the Buycoins API...`

### CODE REVIEW

**Comment by somtochiama** [comment]
Hey @immatobi , Thanks for your PR❤️.
Please limit this pull request to just the changes to readme.
I will review the changes to `tsconfig.json` to the your other pull request.

File: README.md [hunk 1]
```diff
@@ -395,5 +395,5 @@ This project is in continous development and is striving for feature parity with
 
 ### Contributing.
 
-Your pull request are definitely welcome :heart:! We alway looking to add more test and acheive feature parity with the Buycoins API. Take a look at [Contributing.md](Contributing.md)
+Your pull request are definitely welcome :heart:! We're always looking forward to add more test and acheive feature parity with the Buycoins API. Take a look at [Contributing.md](Contributing.md)
```
Comment (somtochiama) [inline README.md:398]:
```suggestion
Your pull requests are definitely welcome :heart:! We're always looking forward to adding more tests and achieving feature parity with the Buycoins API. Take a look at [Contributing.md](Contributing.md)
```

Comment (immatobi) [reply README.md:398]: Thank you for replying @SomtochiAma i will do exactly that. Thank you so much.

**Comment by immatobi** [comment]
I've made the change @SomtochiAma please check the latest commit i made.

**Pull request merged**